compiler/src/test/java/bullet/impl/AbstractComponentProcessorTest.java

```diff
@@ -67,35 +68,35 @@
         "  SimpleSubcomponent simpleSubcomponent();",
         "}");
     JavaFileObject generatedBullet = JavaFileObjects.forSourceLines("test.BulletSimpleComponent",
-        "package test;",
-        "",
-        "import bullet.ObjectGraph;",
-        "import java.lang.Class;",
-        "import java.lang.IllegalArgumentException;",
-        "import java.lang.Override;",
-        "import javax.annotation.Generated;",
-        "",
-        "@Generated(\"bullet.impl.ComponentProcessor\")",
-        "public final class BulletSimpleComponent implements ObjectGraph {",
-        "  private final SimpleComponent component;",
-        "",
-        "  public BulletSimpleComponent(final SimpleComponent component) {",
-        "    this.component = component;",
-        "  }",
-        "",
-        "  @Override",
-        "  public <T> T get(final Class<T> type) {",
-        "    if (type == SomeInjectableType.class) {",
-        "      return type.cast(this.component.someInjectableType());",
-        "    }",
-        "    throw new IllegalArgumentException()",
-        "  }",
-        "",
-        "  @Override",
-        "  public <T> T inject(final T instance) {",
-        "    throw new IllegalArgumentException();",
-        "  }",
-        "}");
+        "package test;\n" +
```

Comment:
I hand-wrote the code here. If you just copy/paste the actual produced code, you're not actually testing whether it complies with what you expect.